ttnn/cpp/ttnn/deprecated/tt_dnn/op_library/compute_kernel_config.cpp

```diff
@@ -0,0 +1,110 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```

Comment:
Use 2024?

Reply:
I’ll update whole codebase in a separate PR

Reply:
I think we just use the current year in the header for new files, don't want to update the entire codebase to the current year? @tt-rkim can you confirm?

Reply:
Thank you, I’d like to see some clarification. 

Best case scenario both created and modify time are indicated. 
Typically header tells at least how fresh the change is (so year should be updated on file modification too)
What we got in the codebase consistently is year is not updated because header is copy pasted. 
That’s why I would prefer to fix it once for everything instead of for this specific file. 
if that’s not how you want to approach it, I will make sure to update this file. My concern is that it’s not maintainable.

Reply:
I'm just looking for consistent behaviour, I'm fine with whatever approach we decide. What I was originally told was to use the year the file was created which is what I've been following.

I assume there are actual nuances related to the date for copyright purposes and that there is a "correct" way we should be doing things, so would be good to get clarification and get this broadcasted to the team to clear any confusion and ensure consistent behaviour/expectations.